view/theme/frio/templates/wall_thread.tpl

```diff
@@ -248,7 +248,7 @@ as the value of $top_child_total (this is done at the end of this file)
 		<div class="wall-item-content {{$item.type}}" id="wall-item-content-{{$item.id}}">
 			{{* insert some space if it's an top-level post *}}
 			{{if $item.thread_level==1}}
-			<div style="height:10px;">&nbsp;</div> <!-- use padding/margin instead-->
+			<div id="wall-spacer" style="height:10px;">&nbsp;</div> <!-- use padding/margin instead-->
```

Comment:
As `wall-spacer` now has a name; you could move the styling from the template (`height: 10px;`) to the main CSS file of frio. Your changes can then be appied in plusminus style as planned.